Mainframe3270/x3270.py

```diff
@@ -57,33 +57,51 @@ def open_connection(
         port: int = 23,
         extra_args: Optional[Union[List[str], os.PathLike]] = None,
     ):
-        """Create a connection to IBM3270 mainframe with the default port 23. To make a connection with the mainframe
-        you only must inform the Host. You can pass the Logical Unit Name and the Port as optional.
+        """Create a connection to a IBM3270 mainframe with the default port 23.
+        To make a connection with the mainframe you only need to specify the Hostname.
+        You can pass the Logical Unit Name and the Port as optional.
 
-        If you wish, you can provide further configuration data to ``extra_args``. ``extra_args`` takes in a list,
+        If you wish, you can provide further configuration data via ``extra_args``. ``extra_args`` takes in a list,
         or a path to a file, containing [https://x3270.miraheze.org/wiki/Category:Command-line_options|x3270 command line options].
+
         Entries in the argfile can be on one line or multiple lines. Lines starting with "#" are considered comments.
         | # example_argfile_oneline.txt
-        | --accepthostname myhost.com
+        | -accepthostname myhost.com
 
         | # example_argfile_multiline.txt
-        | --accepthostname myhost.com
+        | -accepthostname myhost.com
         | # this is a comment
-        | --charset french
+        | -charset french
+        | -port 992
 
         Please make sure the arguments you are providing are available for your specific x3270 application and version.
+        For example the subset of [https://x3270.miraheze.org/wiki/Wc3270/Command-line_options|wc3270 line options].
 
         Example:
             | Open Connection | Hostname |
             | Open Connection | Hostname | LU=LUname |
             | Open Connection | Hostname | port=992 |
-            | ${extra_args}   | Create List | --accepthostname | myhost.com | --cafile | ${CURDIR}/cafile.crt |
+            | ${extra_args}   | Create List | -accepthostname | myhost.com | -cafile | ${CURDIR}/cafile.crt |
+            } append to list  | ${extra_args} | -port | 992 |
             | Open Connection | Hostname | extra_args=${extra_args} |
             | Open Connection | Hostname | extra_args=${CURDIR}/argfile.txt |
         """
         self.host = host
         self.lu = LU
         self.port = port
+        if extra_args:
+            extra_args = self._process_args(extra_args)
```

Comment:
I think I like this approach of processing the arguments in x3270.py instead of py3270.py. I think it would be safe to remove the `ExecutableApp` superclass in py3270.py altogether.